comp/api/api/utils/common.go

```diff
@@ -7,20 +7,12 @@
 package utils
 
 import (
-	"encoding/json"
 	"net"
 	"net/http"
 
 	grpccontext "github.com/DataDog/datadog-agent/pkg/util/grpc/context"
 )
 
-// SetJSONError writes a server error as JSON with the correct http error code
```

Comment:
Since we are modifying this file, should we rename it to `connection.go` or `grpc.go` (As it only has a utility function for grpc connection) ?